python/bindings/include/openravepy/openravepy_jointinfo.h

```diff
@@ -36,12 +36,20 @@ class PySideWall
     int type = 0;
 };
 
+class PyTestPickle
```

Comment:
@tgn3000 is this class necessary?

Reply:
no, these were merged from `pybind11_stable` branch by mistake. Removed.